Build/Common.proj

```diff
@@ -59,8 +59,7 @@
 
   <PropertyGroup Condition="'$(ReferencedPlatform)' == ''">
     <ReferencedPlatform>net45</ReferencedPlatform>
```

Comment:
I was planning on eventually getting rid of it, but it's somewhat unrelated to the .NET Core support so I was going to do it as a separate PR.